OF fails to build (Linux Mint 13)

```
chessguy3@bryce-virtual-machine-mint ~/openFrameworks/scripts/linux $ ./compileOF.sh 
HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-0.10 gstreamer-0.10 gstreamer-video-0.10 gstreamer-base-0.10 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-0.10 gstreamer-0.10 gstreamer-video-0.10 gstreamer-base-0.10 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
Compiling ../../../libs/openFrameworks/video/ofGstUtils.cpp
g++ -g3 -Wall -std=c++0x -march=native -mtune=native -DOF_USING_GTK -DOF_USING_MPG123 -pthread -D_REENTRANT -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12 -I/usr/include/gstreamer-0.10 -I/usr/include/libxml2 -I/usr/include/alsa -I/usr/include/GL -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/atk-1.0 -I/usr/include/gdk-pixbuf-2.0 -I../../../libs/cairo/include -I../../../libs/cairo/include/pixman-1 -I../../../libs/cairo/include/libpng15 -I../../../libs/cairo/include/cairo -I../../../libs/fmodex/include -I../../../libs/glfw/include -I../../../libs/glfw/include/GLFW -I../../../libs/kiss/include -I../../../libs/openssl/include -I../../../libs/openssl/include/openssl -I../../../libs/poco/include -I../../../libs/rtAudio/include -I../../../libs/tess2/include -I../../../libs/openFrameworks -I../../../libs/openFrameworks/graphics -I../../../libs/openFrameworks/types -I../../../libs/openFrameworks/video -I../../../libs/openFrameworks/gl -I../../../libs/openFrameworks/utils -I../../../libs/openFrameworks/events -I../../../libs/openFrameworks/app -I../../../libs/openFrameworks/3d -I../../../libs/openFrameworks/sound -I../../../libs/openFrameworks/communication -I../../../libs/openFrameworks/math -DDEBUG  -MMD -MP -MF ../../../libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/video/ofGstUtils.d -MT../../../libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/video/ofGstUtils.o -o ../../../libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/video/ofGstUtils.o -c ../../../libs/openFrameworks/video/ofGstUtils.cpp
../../../libs/openFrameworks/video/ofGstUtils.cpp:6:36: fatal error: gst/video/gstvideometa.h: No such file or directory
compilation terminated.
make[1]: *** [../../../libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/video/ofGstUtils.o] Error 1
make: *** [Debug] Error 2
there has been a problem compiling Debug OF library
please report this problem in the forums
```

--- a/scripts/linux/fedora/install_dependencies.sh
+++ b/scripts/linux/fedora/install_dependencies.sh
@@ -10,7 +10,7 @@ if [ $EUID != 0 ]; then
    exit 1
 fi
 
-yum install freeglut-devel alsa-lib-devel libXmu-devel libXxf86vm-devel gcc-c++ libraw1394-devel gstreamer-devel gstreamer-plugins-base-devel libudev-devel libtheora-devel libvorbis-devel openal-soft-devel libsndfile-devel python-lxml gstreamer-devel glew-devel flac-devel freeimage-devel cairo-devel pulseaudio-libs-devel openssl-devel libusbx-devel gtk2-devel libXrandr-devel libXi-devel opencv-devel libX11-devel
+yum install freeglut-devel alsa-lib-devel libXmu-devel libXxf86vm-devel gcc-c++ libraw1394-devel gstreamer1-devel gstreamer1-plugins-base-devel libudev-devel libtheora-devel libvorbis-devel openal-soft-devel libsndfile-devel python-lxml glew-devel flac-devel freeimage-devel cairo-devel pulseaudio-libs-devel openssl-devel libusbx-devel gtk2-devel libXrandr-devel libXi-devel opencv-devel libX11-devel
 
 exit_code=$?
 if [ $exit_code != 0 ]; then

